src/cpp/benders/benders_core/include/antares-xpansion/benders/benders_core/StartUp.h

```diff
@@ -12,7 +12,7 @@ namespace Benders {
 class StartUp {
  public:
   bool StudyAlreadyAchievedCriterion(const SimulationOptions& options,
-                                     const Writer& writer,
+                                     const std::shared_ptr<Output::OutputWriter>& writer,
```

Comment:
```suggestion
                                     std::shared_ptr<Output::OutputWriter> writer,
```

Reply:
I went one step further and changed it to a raw pointer. We're using the value, not storing it